doc/index.rst

```diff
@@ -45,6 +52,7 @@ If you are doing research using Blackbird, please cite
     Nathan Killoran, Josh Izaac, Nicolás Quesada, Ville Bergholm, Matthew Amy, and Christian Weedbrook.
     *Strawberry Fields: A Software Platform for Photonic Quantum Computing* 2018. `arXiv:1804.03159 <https://arxiv.org/abs/1804.03159>`_
 
+
```

Comment:
@josh146 pls update reference here as well (seems it was overlooked)

Reply:
Doing it now!